web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -1,25 +1,37 @@
 <script>
-	import { mdiContain	} from "@mdi/js";
+	import { mdiArrowLeft, mdiContain } from "@mdi/js";
 	import { fade } from "svelte/transition";
 	import { logo } from "$lib/dusk/icons";
 	import {
-		Anchor, Badge, Icon, Tooltip
+		Anchor, AnchorButton, Badge, Card, Icon, Throbber, Tooltip
 	} from "$lib/dusk/components";
 	import { createFeeFormatter, createTransferFormatter } from "$lib/dusk/currency";
-	import { settingsStore } from "$lib/stores";
 	import { calculateAdaptiveCharCount, middleEllipsis } from "$lib/dusk/string";
 	import { onMount } from "svelte";
+	import { sortByHeightDesc } from "$lib/transactions";
+	import { compose, take } from "lamb";
 
-	/** @type Transaction[] */
-	export let transactions;
+	/** @type {String} */
+	export let language;
+
+	/** @type {Number | Undefined} */
+	export let transactionLimit = undefined;
```

Comment:
As you're refactoring: is it really necessary to have the `transaction` prefix here and a `transaction` property at line 24?
The component is already named `Transactions`, we know we're talking about transactions.
I would go for `items` in place of `transactions` to keep a similar interface across components that render lists, and simply `limit` (actually, if you find something more explicit it's better) instead of `transactionLimit`.

cc @kieranhall @deuch13

Reply:
I'm in favour of members being named as descriptively AND as tersely as possible.  Therefore, I think that we can lose the prefix since the component is scoped to transactions itself.